readthedocs/doc_builder/config.py

```diff
@@ -59,10 +57,9 @@ def load_yaml_config(version):
             path=checkout_path,
             env_config=env_config,
         )
-    except InvalidConfig:
```

Comment:
It seems we didn't change the logic around this -- why is it being removed?

Reply:
The lines below have the explanation

> # This is a subclass of ConfigError, so has to come first
>  raise

We just basically avoided catching anything subclassing ConfigError first, this was just to capture the exception when a file isn't found. But now we have a specific exception for it.